### PR TITLE
DOCS-8639 Cloud Cost Monitor Update

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do? What is the motivation?

Update the Cost Monitors doc page to include information about Forecast and Anomalies monitors. To circle back in the last week of September for merging it in.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/monitors/types/cloud_cost.md [hunk 1]
```diff
@@ -61,16 +85,43 @@ Any cost type or metric reporting to Datadog is available for monitors. You can
...
+
+{{% /tab %}}
+{{% tab "Threshold" %}}
+
+If you are using the **Cost Threshold** monitor type, you can trigger an alert when the cloud cost is `above`, `below`, `above or equal`, or `below or equal to` a threshold.  
+
+{{% /tab %}}
+{{% tab "Forecast" %}}
+
+If you are using the **Cost Forecast** monitor type, you can trigger an alert when the cloud cost is `above`, `below`, `above or equal`, or `below or equal to`, `equal to`, or `not equal to` a threshold.
```
Comment (jhgilbert) [inline content/en/monitors/types/cloud_cost.md:111]:
```suggestion
If you are using the **Cost Forecast** monitor type, you can trigger an alert when the cloud cost is `above`, `below`, `above or equal`, `below or equal to`, `equal to`, or `not equal to` a threshold.  
```

File: content/en/monitors/types/cloud_cost.md [hunk 2]
```diff
@@ -43,16 +43,40 @@ Optionally, click the **+ Create Monitor** button in the cost report side panel.
...
 
-Choose between a **Compare Costs Over Time** or a **Set Daily Cost Threshold** monitor type.
+You can select from the following monitor types.
+
+{{< tabs >}}
+{{% tab "Changes" %}}
 
 | Cost Type | Description | Usage Examples |
 | ---  | ----------- | ----------- |
 | Cost Changes  | Compare costs on a daily, weekly or monthly basis | Alert when the difference between today's cost and the week before is over 5% |
```
Comment (jhgilbert) [inline content/en/monitors/types/cloud_cost.md:53]: Add periods after the sentences in this table for consistency with your new ones?

File: content/en/monitors/types/cloud_cost.md [hunk 3]
```diff
@@ -43,16 +43,40 @@ Optionally, click the **+ Create Monitor** button in the cost report side panel.
...
+
+{{% /tab %}}
+{{% tab "Threshold" %}}
+| Cost Type | Description | Usage Examples |
+| ---  | ----------- | ----------- |
 | Cost Threshold | Set alerts on total costs exceeding a threshold in a day | Set alerts when today's total cost exceeds $10,000 |
```
Comment (jhgilbert) [inline content/en/monitors/types/cloud_cost.md:59]: Add periods after the sentences in this table for consistency with your new ones?

**Review by jhgilbert — APPROVED** [review]
Approved with very minor suggestions, thanks!

File: content/en/monitors/types/cloud_cost.md (hunk 2 above)
Comment (alai97) [reply content/en/monitors/types/cloud_cost.md:53]:
```suggestion
| Cost Changes  | Compare costs on a daily, weekly or monthly basis | Alert when the difference between today's cost and the week before is over 5%. |
```

File: content/en/monitors/types/cloud_cost.md (hunk 3 above)
Comment (alai97) [reply content/en/monitors/types/cloud_cost.md:59]:
```suggestion
| Cost Threshold | Set alerts on total costs exceeding a threshold in a day | Set alerts when today's total cost exceeds $10,000. |
```

File: content/en/monitors/types/cloud_cost.md [hunk 4]
```diff
@@ -39,12 +39,14 @@ You can also create Cloud Cost monitors from [Cloud Costs Analytics][2]. Click o
...
 
-Choose between a **Compare Costs Over Time** or a **Set Daily Cost Threshold** monitor type.
+You can choose from the following monitor types.
 
 | Cost Type | Description | Usage Examples |
 | ---  | ----------- | ----------- |
 | Cost Changes  | Compare costs on a daily, weekly or monthly basis | Alert when the difference between today's cost and the week before is over 5% |
 | Cost Threshold | Set alerts on total costs exceeding a threshold in a day | Set alerts when today's total cost exceeds $10,000 |
+| Cost Forecast | Set alerts based on forecasted costs exceeding a threshold by the end of the current month. | Alert daily if the forecasted cost for this month is projected to exceed $500. |
+| Cost Anomalies | Detect anomalies by comparing current costs to historical data, using a defined lookback period. | Alert if 3 days from the past 30 days show significant cost anomalies compared to historical data. |
```
[the remark this answers is not in the document]
Comment (alai97) [reply content/en/monitors/types/cloud_cost.md:72]:
```suggestion
| Cost Anomalies | Detect anomalies by comparing current costs to historical data, using a defined lookback period. Incomplete days are excluded from analysis to ensure accuracy. | Alert if 3 days from the past 30 days show significant cost anomalies compared to historical data. |
```

**Comment by alai97** [comment]
Accepted your suggestion, @HemeryJu!

**Pull request merged**